common/schema.h

```diff
@@ -263,6 +263,7 @@ namespace swss {
 #define CFG_LAG_INTF_TABLE_NAME           "PORTCHANNEL_INTERFACE"
 #define CFG_VLAN_INTF_TABLE_NAME          "VLAN_INTERFACE"
 #define CFG_VLAN_SUB_INTF_TABLE_NAME      "VLAN_SUB_INTERFACE"
+#define CFG_MGMT_PORT_TABLE_NAME          "MGMT_PORT"
```

Comment:
This line is not related to log message config

Reply:
Yes, it doesn't.
That is why I put it in a separate commit.

Reply:
The PR title is "Login messages config". SONiC community requires to divide different features contribution into different PRs.